cli/project/dockerfiles.go

```diff
@@ -181,11 +183,40 @@ func getInstallTarantoolLayers(ctx *context.Ctx) (string, error) {
 	return installTarantoolLayers, nil
 }
 
+func getOpenSourceInstallerLink(version string) string {
+	parts := strings.SplitN(version, ".", 3)
+	major, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return ""
+	}
+
+	minor, err := strconv.Atoi(parts[1])
+	if err != nil {
+		return ""
+	}
+
+	if major >= 2 && minor > 8 {
```

Comment:
3.0.0 will not match with this condition.

Reply:
Fixed